src/components/App/userReducer.ts

```diff
@@ -0,0 +1,29 @@
+import { JWTToken } from '../../../types/JWT'
+import { SET_TOKEN } from '../../actions/types'
+
+const initialState = {
+    user: { token: '' },
+}
+
+export interface User {
+    token: JWTToken
```

Comment:
shoudll'nt the userId be also stored there, as well as the login state?

Reply:
I work on it 
Check in my pr
https://github.com/Instant-Visio/InstantVisio-WebApp/pull/330/commits/f71a540879b7a7c0b53a566610488c6b25f90fdd

Reply:
Sorry, I forgot to assign myself [the issue](https://github.com/Instant-Visio/InstantVisio-WebApp/issues/334), and did not see the notification of your assignment. But wasn't your PR only meant to address `Redirect admin dashboard` ?